Jenkinsfile

```diff
@@ -77,15 +77,6 @@ node ('master') {
             sh './bin/run_tests'
         }
 
-        stage("Create git archive") {
-            sh '''
-                REPO=$(git remote show -n origin | grep Fetch | awk -F'[/.]' '{print $6}')
-                VERSION=`git describe --dirty`
-                git archive HEAD --format=zip -9 --output=$REPO-$VERSION.zip
-                git archive HEAD --format=tgz -9 --output=$REPO-$VERSION.tgz
-            '''
-        }
-
```

Comment:
Did you add a jira or some way to indicate this should be re-added?

Reply:
Created story: https://jira.hyperledger.org/browse/STL-1127